internal/service/ecs/task_definition_test.go

```diff
@@ -167,6 +167,62 @@ func TestAccECSTaskDefinition_withDockerVolumeMinimal(t *testing.T) {
 	})
 }
 
+func TestAccECSTaskDefinition_withRuntimePlatform(t *testing.T) {
+	var def ecs.TaskDefinition
+
+	tdName := sdkacctest.RandomWithPrefix("tf-acc-td-with-runtime-platform")
+	resourceName := "aws_ecs_task_definition.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acctest.PreCheck(t) },
+		ErrorCheck:   acctest.ErrorCheck(t, ecs.EndpointsID),
+		Providers:    acctest.Providers,
+		CheckDestroy: testAccCheckTaskDefinitionDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccTaskDefinitionWithRuntimePlatformMinimalConfig(tdName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckTaskDefinitionExists(resourceName, &def),
```

Comment:
lets add checks for the values  for example: `resource.TestCheckResourceAttr(resourceName, "runtime_platform.#", "1"),`